### ISSUE TITLE
It would be nice to derive Default for message structs

### ISSUE THREAD

**Issue by timthelion** [danburkert/prost#136]
I see that this has already been addressed in https://github.com/danburkert/prost/pull/117 . I'm ending up with a lot of boring code like:

````
pub fn blank_cell_runtime() -> gradesta::CellRuntime {
 gradesta::CellRuntime {
  cell: None,
  update_count: None,
  click_count: None,
  deleted: None,
  creation_id: None,
  cell_runtime_modes: HashMap::new(),
  cell_modes: HashMap::new(),
  link_modes: HashMap::new(),
  for_link_modes: HashMap::new(),
  back_link_modes: HashMap::new(),
 }
}

pub const blank_actor_metadata: gradesta::ActorMetadata = gradesta::ActorMetadata {
 name: None,
 source_url: None,
 privacy_policy: None,
};
````

So that I can send messages with only a few fields set.

**Comment by danburkert** [comment]
Not sure I understand the question fully.  All generated message types do implement `Default`, so the implementation of that method and const could simply be `Default::default()`.  I don’t see a connection between your question and unknown fields, could you elaborate?

**Comment by timthelion** [comment]
I did not know that. I read the discussion here: https://github.com/danburkert/prost/issues/2#issuecomment-402772743 in which @per-gron writes "This would work if all generated structs would #[derive(Default)] (or equivalent), which seems like a sensible thing to do in general because Protobufs in general assume that no code breaks from just adding a field." And I assumed that default was NOT derived. Further, I saw code in that PR https://github.com/danburkert/prost/pull/117/files#diff-6d0f1e2a66e5c7be23fcd64650dc1044R3 which appeared to be testing the addition of default handling, though I admit that now I am not sure where the new default handling code is in that PR which is to be tested.
